api/URLSearchParams.json

```
@@ -173,6 +217,57 @@
               "deprecated": false
             }
           }
+        },
+        "sequence": {
+          "__compat": {
+            "description": "sequence for <code>init</code> object",
+            "support": {
+              "chrome": {
+                "version_added": "58"
+              },
+              "chrome_android": {
+                "version_added": "58"
+              },
+              "edge": {
+                "version_added": "17"
+              },
+              "firefox": {
+                "version_added": "53"
+              },
+              "firefox_android": {
+                "version_added": "53"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": "7.10.0"
+              },
+              "opera": {
+                "version_added": "45"
+              },
+              "opera_android": {
+                "version_added": "45"
+              },
+              "safari": {
+                "version_added": "11"
+              },
+              "safari_ios": {
+                "version_added": "11"
+              },
+              "samsunginternet_android": {
+                "version_added": "7.2"
+              },
+              "webview_android": {
+                "version_added": "61"
```

To sync up with Chrome and Chrome for Android

```suggestion
                "version_added": "58"
```